urwid/event_loop/asyncio_loop.py

```diff
@@ -65,6 +67,9 @@ def __init__(self, *, loop: asyncio.AbstractEventLoop | None = None, **kwargs) -
         if loop:
             self._loop: asyncio.AbstractEventLoop = loop
         else:
+            if IS_WINDOWS:
+                asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
setting a global policy as a side-effect of creating an instance of AsyncioEventLoop is surprising. At a minimum this should be warned about in the class docstring

Reply:
Added log record and restore of default in destructor if changed